packages/v4/src/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -3,5 +3,92 @@ id: Tree view
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/get-in-touch)
+A **tree view** is a structure that displays data in a hierarchical view. It can be used in a [dropdown](/components/dropdown), [drawer](/components/drawer), [primary-detail](/demos/primary-detail), [modal](/components/modal), or [wizard](/components/wizard).
+
+A default tree view provides a tree structure with items and an arrow to expand or collapse child items. Every tree view can be extended with these [optional elements](#tree-view-capabilities).
+
+## Tree view elements
+
+A default tree view consists of 5 elements:
+
+<img src="./img/tree-view-elements.png" width="332" />
+
+1. **Expand/collapse button:** A button that allows you to expand or collapse child nodes 
```

Comment:
I actually don't know if I'd use the word "button" here, because the whole area of the item is clickable to expand, not just the arrow. Also isn't it expanding the parent, and not the children? So maybe change it to something like:
"Expand/colla[se: Expand or collapse parents node to reveal child nodes"

Reply:
Makes sense! 👍 I just recommend that we keep each line consistent in terms of part of speech. The other options start with a definition ("A node that..," or "The last item..."), so we'd want to keep the same thing going here instead of giving an action. Perhaps "An option to expand or collapse parent nodes to reveal child nodes" could work?

Reply:
It sounds good to me.